packages/api/controllers/nomination.js

```diff
@@ -178,15 +178,21 @@ const syncNominations = async (req, res) => {
 const emailVerification = async (req, res) => {
   try {
     const { token } = req.params;
-    const { nomination: id } = jwt.verify(token, process.env.JWT_SECRET);
+    const { data: id } = jwt.verify(token, process.env.JWT_SECRET);
+    console.log(id, '-----nomination------');
```

Comment:
Looks like a stray comment here.

Reply:
Ahh, okay. I forgot to remove those. Got it!